theme/templates/resource-landing-page/citation.html

```diff
@@ -2,22 +2,31 @@
 
 <div class="col-sm-12 content-block">
     <span id="citation" class="page-anchor"></span>
-    <h3>How to Cite</h3>
+    <h3>How to Cite
 
+    </h3>
     <div class="row">
         <div class="col-xs-12">
             <div class="flex">
-                <div id="citation-text">{{ citation|urlize }}</div>
-
-                <button class="btn btn-default clipboard-copy" data-target="citation-text"
-                        type="button">
+                {% if custom_citation %}
+                    <div id="citation-text">{{ custom_citation|urlize }}</div>
+                {% else %}
+                    <div id="citation-text">{{ citation|urlize }}</div>
+                {% endif %}
+                <span id="edit-citation-control" style="display: none" data-toggle="modal" data-placement="auto" class="glyphicon glyphicon-pencil icon-blue icon-button table-icon"
+                      title="Edit citation" data-target="#customize-citation-dialog">
```

Comment:
I believe we already have a css element of ```display: none``` style defined in the css file so you can use it directly rather than embedding the style in the template file.

Reply:
This is a more clear convention, especially when using jQuery/Vue to modify the attribute.